mmcv/fileio/file_client.py

```diff
@@ -108,14 +109,15 @@ class PetrelBackend(BaseStorageBackend):
 
     def __init__(self,
                  path_mapping: Optional[dict] = None,
+                 conf_path: str = None,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.